package.json

```diff
@@ -81,17 +81,17 @@
     "postcss-import": "^8.1.2",
     "postcss-url": "^5.1.2",
     "requirejs": "^2.2.0",
-    "rollup": "^0.26.3",
-    "rollup-plugin-babel": "^2.4.0",
+    "rollup": "^0.26.7",
+    "rollup-plugin-babel": "^2.5.1",
     "rollup-plugin-includepaths": "^0.1.5",
     "rollup-plugin-uglify": "^0.3.1",
-    "run-sequence": "^1.2.0",
+    "run-sequence": "^1.2.1",
     "rxjs": "^5.0.0-beta.8",
     "selenium-webdriver": "^2.53.2",
     "sinon": "^1.17.4",
     "sinon-chai": "^2.8.0",
     "stylelint": "^6.5.1",
     "stylelint-config-standard": "^8.0.0",
-    "uglify-js": "mishoo/UglifyJS2#harmony"
+    "uglify-js": "github:mishoo/UglifyJS2#harmony"
```

Comment:
Nothing at all, but I ran a npm update or install and it normalised this repo for me, so I thought I'd keep it.